src/main/resources/openapi.json

```diff
@@ -753,6 +753,56 @@
                 }
             ]
         },
+        "/create-topic/{topicname}": {
```

Comment:
I don't think this API design is going to work. What are you going to have if we want the topic deletion, a new `/delete-topic` endpoint? It's not how REST API design works and scales.
I think we would need a new more general `/admin` endpoint? So a corresponding `/admin/topics` on which we can use the usual POST to create, DELETE to delete and so on? I think you should think more broader than just adding the topic creation, in terms of the API evolution for additional operations.

Reply:
You're right, I thought about it but couldn't decide, I will do as you said.